docs/admin_case_study.md

```diff
@@ -0,0 +1,371 @@
+# Admin case study
+
+This document provides several case studies of `pg_diffix` configuration. The examples are from real databases that are used for the Diffix bounty programs. The databases can be found at https://db001.gda-score.org/.
+
+## The banking database
+
+The banking database is taken from a dataset originally used in a machine learning competition. The data itself is from a Czech bank, and had been pseudonymized to protect privacy. The database is named `banking0`. It has seven tables: `accounts`, `cards`, `clients`, `disp`, `loans`, `orders`, and `transactions`.
+
+### Database preparation
```

Comment:
This heading makes it sound like the section will instruct the user how to prepare the database, but in reality it is only describing its current state.